protocol-configuration/dependencies.ts

```diff
@@ -158,6 +162,9 @@ const dependencies: DependencyMap = {
   daiPSMFeiSkimmer: {
```

Comment:
daiPSMFeiSkimmer should be remvoed from deps file

Reply:
fixed